### PR TITLE
Created fixtures for MenuItemReview

### LINKED ISSUE #36: Create fixtures for `MenuItemReview`
# Dependencies

None; this issue should be done first before other issues for the
MenuItemReview table

# Acceptance Criteria:

- [x] In the frontend directory `frontend/src/main/fixtures` there is a file called 
      `menuItemReviewFixtures.js` that contains Javascript fixtures
      for the database table that is similar to the `restaurantFixtures.js` 
      and `ucsbDatesFixtures.js` files in the same directory.
- [x] Each example should correspond to the JSON format of an object returned by 
      the GET API endpoint from team02 for the `MenuItemReview` database table object.

# What to do next

** Make a PR**: When you've met all of the acceptance criteria, do a PR for this branch,
being sure to enter a [reasonable title](https://ucsb-cs156.github.io/topics/pull_requests.html#pr-titles)
and [description](https://ucsb-cs156.github.io/topics/pull_requests.html#pr-descriptions).

In the description, include the text "Closes #n" where `n` is the number of
this issue.

Then ask for a team member to review your PR, and drag this issue from "In Progress" to "In Review".

** Review Others' PRs**: Next, see if there are any PRs that need to be code reviewed and merged.  Unless you are the
first person on your team to complete a PR, there likely is at least one.
See: 
* [General Code Review Advice](https://ucsb-cs156.github.io/topics/code_reviews/)
* [Performin Code Reviews on Github](https://ucsb-cs156.github.io/topics/code_reviews/code_reviews_github.html)

** Start your next issue **: Then, find the issue for "Form Component for `MenuItemReview` plus tests" on the Kanban board in the ToDo column.

* Assign it to yourself
* Drag into "In Progress"
* Create a new branch, starting from the branch for the issue you just finished.
* Start on the new issue.

### PR DESCRIPTION
**Pull request by bella-ward**
Created fixtures for MenuItemReview. 

Closes #36

### CODE REVIEW

File: frontend/src/fixtures/menuItemReviewFixtures.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    oneItemReview: {
+        "id": 1, 
+        "itemId": 14,
+        "reviewerEmail": "ward@ucsb.edu",
+        "stars": 5,
+        "dateReviewed": "2022-01-02T12:00:00",
+        "comments": "I love the chocolate cake!"
+        
+    },
+    threeDates: [
```
Comment (YicongYan) [inline frontend/src/fixtures/menuItemReviewFixtures.js]: should be changed to "threeItemReviews"

**Review by YicongYan — CHANGES_REQUESTED** [review]
line 11 in menuItemReviewFixtures.js should be changed to "threeItemReviews"

File: frontend/src/fixtures/menuItemReviewFixtures.js (hunk 1 above)
Comment (bella-ward) [reply frontend/src/fixtures/menuItemReviewFixtures.js]: Ahh, thanks  changed it.

**Review by YicongYan — APPROVED** [review]
Looks good to me now!

**Review by YicongYan — CHANGES_REQUESTED** [review]
OK, the Fixture looks good to me, but it seems like you also contain a MenuitemItemForm.js without a test, and that causes some test fails on Github. I also noticed that some new lines are added to package-lock.json, so it seems dangerous to me to merge it. I will need to talk with a few other team members about it. **But please don't worry about it, because I'm still not sure whether it's a problem. If it is, we will fix it as a team. it's not going to be a big problem, so plz don't worry**

**Comment by YicongYan** [comment]
Looks good to me!

**Pull request merged**